docs/src/pages/[platform]/components/placeholder/examples/PlaceholderThemeExample.tsx

```diff
@@ -1,12 +1,17 @@
-import { Theme, Placeholder, ThemeProvider } from '@aws-amplify/ui-react';
+import { Placeholder, ThemeProvider, Theme } from '@aws-amplify/ui-react';
 
 const theme: Theme = {
   name: 'placeholder-theme',
   tokens: {
     components: {
       placeholder: {
-        startColor: { value: 'rebeccapurple' },
-        endColor: { value: 'cyan' },
+        transitionDuration: { value: '1250ms' },
+        startColor: { value: '{colors.purple.40}' },
+        endColor: { value: '{colors.purple.100}' },
```

Comment:
I feel like `purple.100` is a bit heavy almost close to black. Maybe `purple.80` is a better choice?